### PR TITLE
From Posts to Reports: Leveraging LLMs for Social Media Data Mining

### PR DESCRIPTION
**Pull request by vladadu**
article code

### CODE REVIEW

File: articles/machine_learning_engineering/experimental_trackers_efficiency_gradio_vs_wandb_training_diffusion_models/Diffusion_Model_Training_Gradio.ipynb
Comment (iusztinpaul) [inline articles/machine_learning_engineering/experimental_trackers_efficiency_gradio_vs_wandb_training_diffusion_models/Diffusion_Model_Training_Gradio.ipynb:1]: Could you exclude these two files (the Notebooks under the MLE folder) from the PR please?

File: articles/generative_ai/data_extraction_from_social_media_posts_using_llms/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+
+## Usage
+
+To start the hole process you have to run the following: 
+
+```shell
+make run
+```
```
Comment (iusztinpaul) [inline articles/generative_ai/data_extraction_from_social_media_posts_using_llms/README.md:56]: Could you also add a sentence about on the env vars you have to fill out?

Something like: `cp .env.example .env` and fill out the env vars from there. We have to assume people don't know how to use `.env` files (we are working with many data scientists 🧑‍🔬)

File: articles/machine_learning_engineering/experimental_trackers_efficiency_gradio_vs_wandb_training_diffusion_models/Diffusion_Model_Training_Gradio.ipynb
Comment (iusztinpaul) [reply articles/machine_learning_engineering/experimental_trackers_efficiency_gradio_vs_wandb_training_diffusion_models/Diffusion_Model_Training_Gradio.ipynb:1]: Or at least explain why these files are changed, as I cannot see the Diff within a Notebook.

Comment (vladadu) [reply articles/machine_learning_engineering/experimental_trackers_efficiency_gradio_vs_wandb_training_diffusion_models/Diffusion_Model_Training_Gradio.ipynb:1]: idk what happened here i just looked over other projects to see how they look and i just opened the files but didn't change anything, so idk why they got in this pr :))

**Review by iusztinpaul — APPROVED** [review]

**Pull request merged**